### ISSUE TITLE
iron-dropdown not aligned properly when used in paper-dropdown-menu and paper-menu-button

### ISSUE THREAD

**Issue by priyankapothala** [PolymerElements/iron-dropdown#80]
### Description
iron-dropdown occupies the entire page when used in paper-dropdown-menu.

### Expected outcome

It should take the max-width of paper-item

### Actual outcome

It is occupying the entire page

### Steps to reproduce
Add paper-dropdown-menu element and try to select a paper-item.

``` html
<paper-dropdown-menu label="Name">
      <paper-menu class="dropdown-content">
           <paper-item>John</paper-item>
           <paper-item>Jack</paper-item>
           <paper-item>Jill</paper-item>
       </paper-menu>
</paper-dropdown-menu>
```
### Browsers Affected

- [x] Chrome
- [x] Firefox
- [x] Safari 9
- [x] Safari 8
- [x] Safari 7
- [x] Edge
- [x] IE 11
- [x] IE 10

**Comment by cacerrillos** [comment]
I'm having the same problem, the bug is actually in [https://github.com/PolymerElements/iron-fit-behavior](url), I downgraded from the recently released 1.1.0 to 1.0.6 and the issue went away.

**Comment by priyankapothala** [comment]
Thanks. I will raise the issue in iron-fit-behavior

**Comment by MajorBreakfast** [comment]
@cacerrillos Thanks for the advice. Downgrading helped as a quick-fix:)

**Comment by cacerrillos** [comment]
It's already been patched :tada: in tag 1.1.1

**Comment by valdrinkoshi** [comment]
Closing this as fixed.
